server/chat-plugins/helptickets.js

```diff
@@ -807,8 +849,15 @@ let commands = {
 			};
 			/** @type {{[k: string]: string}} */
 			const contexts = {
-				'Battle Harassment': 'Please post a link or replay in chat so global staff can check immediately after they join.',
-				'Inappropriate Pokemon Nicknames': 'Please post a link or replay in chat so global staff can check immediately after they join.',
+				'PM Harassment': `Hi! Who was harassing you in private messages?`,
+				'Battle Harassment': `Hi! Who was harassing you, and in which battle did it happen? Please post a link to the battle or a replay of the battle.`,
+				'Inappropriate Username': `Hi! Tell us the username that is inappropriate.`,
+				'Inappropriate Pokemon Nicknames': `Hi! Which user has pokemon with inappropriate nicknames, and in which battle? Please post a link to the battle or a replay of the battle.`,
+				'Room Owner Complaint': `Hi! Which Room Owner are you reporting, and why are you reporting them?`,
+				'Global Staff Complaint': `Hi! Which Global Staff member are you reporting, and why are you reporting them?`,
+				'Appeal': `Hi! Can you please explain why you feel your punishment is undeserved?`,
+				'Public Room Assistance Request': `Hi! A Global Staff member should be here in about a minute to help you.`,
```

Comment:
We might as well ask what the request is? Or is this a request for staff to be in the room? Either way, it's not going to show up on the ticket list without a message.

Reply:
Yeah thats true, I should probably just remove this message completely.

Reply:
Or actually, I'll specify for them to tell us which room it is.